blog/temporal-release/index.md

```diff
@@ -0,0 +1,385 @@
+timezone_provider = { version = "0.0.17", features = ["tzif"] }
+temporal_rs = { version = "0.1.0", no-default-features = true, features = ["sys"]}
+```
+
+We include `sys` as a `temporal_rs` feature to have access to the system
+for the default `Now` implementation. We provide the `tzif` feature to
+`timezone_provider` for the feature gate on `FsTzdbProvider`.
+
+Please note: `timezone_provider` is still considered unstable for the
+near future.
+
+### Some examples
+
+The below examples will be using `temporal_rs` with the default
+features.
+
+#### Retrieve today's date
+
+```rust
+use temporal_rs::Temporal;
+
+// Get today's date
+let today = Temporal::now().to_plain_date_iso(None).unwrap()
+```
+
+#### Date operation's available
+
+Temporal provides a nice API for working with date and date/time via
+`PlainDate` and `PlainDateTime`.
+
+```rust
+use std::convert::TryFrom;
+use temporal_rs::{Calendar, Temporal, options::DifferenceSettings, partial::PartialDuration};
+
+// Get today's date
+let today = Temporal::now().plain_date_iso(None).unwrap();
+
+// We can add a Duration.
+let partial = PartialDuration::empty().with_days(1);
+let tomorrow = today.add(&partial.try_into().unwrap(), None).unwrap();
+
+// We can get the difference two dates
+let diff = today
+    .since(&tomorrow, DifferenceSettings::default())
+    .unwrap();
+
+// We can change the calendar
+let tomorrow_japanese = tomorrow.with_calendar(Calendar::JAPANESE);
+
+// We can retrieve the calendar's RFC9557 string
+println!("{tomorrow_japanese}");
+```
+
+#### Working with dates and time zones
+
+You can also easily work with dates and time zones with the
+`ZonedDateTime` type.
+
+```rust
+use temporal_rs::options::{DifferenceSettings, Disambiguation, OffsetDisambiguation, Unit};
+use temporal_rs::{Calendar, Temporal, TimeZone, ZonedDateTime};
+
+// Parse a ZonedDateTime from utf8 bytes.
+let zdt = ZonedDateTime::from_utf8(
+    b"2025-03-01T11:16:10Z[America/Chicago][u-ca=iso8601]",
+    Disambiguation::Compatible,
+    OffsetDisambiguation::Reject,
+)
+.unwrap();
+
+// Change the time zone.
+let zurich_zone = TimeZone::try_from_str("Europe/Zurich").unwrap();
+let _zdt_zurich = zdt.with_timezone(zurich_zone).unwrap();
+
+// Or get the current ZonedDateTime
+let today = Temporal::now().zoned_date_time_iso(None).unwrap();
+
+// Difference the two `ZonedDateTime`s
+let mut options = DifferenceSettings::default();
+options.largest_unit = Some(Unit::Year);
+let diff = today.since(&zdt, options).unwrap();
+println!("{diff}");
+
+// Change the calendar
+let today_coptic = today.with_calendar(Calendar::COPTIC);
+println!("{today_coptic}");
+```
+
+While we can extend these examples further, a more fun exercise for the
+reader would be taking a look at the
+[Temporal cookbook](https://tc39.es/proposal-temporal/docs/cookbook.html)
+as it displays the utility of the Temporal API using JavaScript and all
+of these examples are now usable from Rust as well.
+
+## FFI and engine adoption
```

Comment:
I wonder if we want a visual in this section to show the engines and their conformance, or maybe its fine?